lib/frinkiac/screencap.rb

```diff
@@ -18,6 +19,20 @@ def image_url
       "#{SITE_URL}/img/#{episode}/#{timestamp}.jpg"
     end
 
+    def meme_url(caption = nil)
+      caption = self.caption if caption.nil?
+      caption = caption.join("\n") if caption.is_a?(Array)
+      "#{SITE_URL}/meme/#{episode}/#{timestamp}.jpg?lines=#{URI.escape caption}"
+    end
+
+    def caption
+      response = Faraday.get("#{CAPTION_URL}?e=#{@episode}&t=#{@timestamp}")
```

Comment:
Thoughts on memoizing the value returned from the API so as not to hit the API every time you call caption on the screencap?

```
def caption
  @caption ||= begin
    response = Faraday.get("#{CAPTION_URL}?e=#{@episode}&t=#{@timestamp}")
    body = JSON.parse(response.body)
      body["Subtitles"].map { |s| s["Content"] }.join("\n")
    end
  end
```